lib/game_play.rb

```diff
@@ -0,0 +1,120 @@
+require './lib/player'
+require './lib/board'
+require './lib/cell'
+require 'pry'
+
+class GamePlay
+
+  attr_reader :human,
+              :computer
+
+  def initialize(human, computer)
+    @human = human
+    @computer = computer
+    @already_fired_upon_coordinates_human = []
+    @already_fired_upon_coordinates_computer = []
+  end
+
+
+    def display_boards
+      sleep(1)
+      puts "=============COMPUTER BOARD============="
+      puts @computer.board.render
+      puts "==============PLAYER BOARD=============="
+      puts @human.board.render(true)
+    end
+
+    def player_chooses_coordinate
```

Comment:
I think having chosen the title of "game_play", choosing a coordinate makes sense to keep in this class. That said, if we think about how the analog would be IRL, it might make more sense to move the choosing coordinates into the player class as well. I'm a little torn on this one--I want to make the player class do more than it already is, but I'm not sure what functionality we could realistically moving.